topology/National Center for High-performance Computing/NCHC/NCHC-CMS-ITP.yaml

```
@@ -0,0 +1,44 @@
+Production: false
+SupportCenter: GGUS
+
+GroupDescription: Opportunistic resource from CMS project.
+GroupID: 1349
+
+Resources:
+  NCHC-CMS-CE:
+    Active: false
+    Description: Opportunistic resource from CMS project.
+    ID: 1433
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Chun-Yu Lin
+          ID: 383ae4982bef24cc546b47c4e4f2e52b57e32da6
+
+      Security Contact:
+        Primary:
+          Name: Chun-Yu Lin
+          ID: 383ae4982bef24cc546b47c4e4f2e52b57e32da6
+
+    FQDN: ui.grid.nchc.org.tw
+
+    Services:
+      CE:
+        Description: HTCondor-CE
+
+    VOOwnership:
+      OSG: 100
+    #  CMS: 100
```

```suggestion
    VOOwnership:
      LIGO: 100
    #  CMS: 100
```

@chunyulin for historical reasons, we refer to the IGWN project as LIGO. I assume that's what you want instead of "OSG"! As far as I know, the OSG doesn't own pay for any of the resources at your site!

---

Indeed. Thanks.